filament/src/details/View.cpp

```diff
@@ -385,6 +385,7 @@ void FView::prepareLighting(FEngine& engine, FEngine::DriverApi& driver, ArenaSc
     }
 
     mPerViewUniforms.prepareAmbientLight(engine, *ibl, intensity, exposure);
+    mPerViewUniforms.prepareIblLight(ibl->getIblOptions());
```

Comment:
Could you fold those two functions instead? prepareAmbientLight() gets a reference to the IBL and has access to IblOptions. There's no need for a second, separate function.

Reply:
Agreed we only need `prepareAmbientLight`.

Reply:
Fixed in 4cf1c080a6ba99c26eccebabd0426674d0eff062